kobo/apps/subsequences/tests/test_submission_extras_api_post.py

```diff
@@ -251,3 +255,72 @@ def test_change_language_list(self):
             },
         }
         # validate(package, schema)
+
+
+class DummyOperation():
+    name = ""
+
+class DummySpeechResult():
+    @property
+    def operation(self):
+        return DummyOperation()
```

Comment:
Needed to prevent pickle error on cache set. Wonder if this could be done more concisely with Magic Mock but return a real object.